js/packages/web/src/components/Banner/index.tsx

```diff
@@ -1,140 +1,167 @@
-            </a>
-          ) : (ownerAddress === wallet.publicKey?.toBase58() && (
+
+      <div className="logo-wrapper">
+        <Link to="/" id="metaplex-header-logo">
+          <img src={logo || ''} />
+        </Link>
+      </div>
+
+      <div id="metaplex-banner-hero">
+        <h1>{headingText}</h1>
+        {subHeadingText && <Text>{subHeadingText}</Text>}
+        {twitterVerification ? (
+          <a
+            href={'https://twitter.com/' + twitterVerification}
+            target="_blank"
+            rel="noreferrer"
+            className="twitter-button"
+          >
+            {' '}
+            <Button shape="round" icon={<TwitterOutlined />}>
+              @{twitterVerification}
+            </Button>
+          </a>
+        ) : (
+          ownerAddress === wallet.publicKey?.toBase58() && (
             <Alert
               className="metaplex-flex-align-items-center metaplex-align-left"
               message="Connect your Twitter account"
-              description={<>Help protect collectors by connecting your store to a Twitter page on <a href="https://naming.bonfida.org/#/twitter-registration" rel="noreferrer" target="_blank">Bonfida</a></>}
+              description={
+                <>
+                  Help protect collectors by connecting your store to a Twitter
+                  page on{' '}
+                  <a
+                    href="https://naming.bonfida.org/#/twitter-registration"
+                    rel="noreferrer"
+                    target="_blank"
+                  >
+                    Bonfida
+                  </a>
+                </>
+              }
               icon={<TwitterOutlined />}
               showIcon
             />
-          ))}
-          {hasSocial && (
-            <Space
-              align="center"
-              direction="horizontal"
-              id="metaplex-banner-hero-social"
-            >
-              {social?.discord && (
-                <a href={social.discord}>
-                  <DiscordIcon />
-                </a>
-              )}
-              {social?.website && (
-                <a href={social.website}>
-                  <WebIcon />
-                </a>
-              )}
-            </Space>
-          )}
-        </Space>
-      </Space>
+          )
+        )}
+        {/* {hasSocial && (
+          <Space
+            align="center"
+            direction="horizontal"
+            id="metaplex-banner-hero-social"
+          >
+            {social?.discord && (
+              <a href={social.discord}>
+                <DiscordIcon />
+              </a>
+            )}
+            {social?.twitter && (
+              <a href={social.twitter}>{<TwitterIcon />}</a>
+            )}
+            {social?.website && (
+              <a href={social.website}>
+                <WebIcon />
+              </a>
+            )}
+          </Space>
+        )} */}
```

Comment:
Yeah lets delete the social since its not supported yet. We can reference these previous commits when we bring it back.